### ISSUE TITLE
'module' object has no attribute 'Default' [Pip Installation, Ubuntu 14.04, cuda 8.0, cudnn 5.1]

### ISSUE THREAD

**Issue by CTTC** [tensorflow/tensorflow#5423]
I just use **Pip Installation** method to install tensorflow 0.11 on ubuntu 14.04. And I installed cuda8.0, cudnn 5.1. The installation went well, but when I tried to import tensorflow in python, the following error occurred.
```bash
>>> import tensorflow
I tensorflow/stream_executor/dso_loader.cc:111] successfully opened CUDA library libcublas.so locally
I tensorflow/stream_executor/dso_loader.cc:111] successfully opened CUDA library libcudnn.so locally
I tensorflow/stream_executor/dso_loader.cc:111] successfully opened CUDA library libcufft.so locally
I tensorflow/stream_executor/dso_loader.cc:111] successfully opened CUDA library libcuda.so.1 locally
I tensorflow/stream_executor/dso_loader.cc:111] successfully opened CUDA library libcurand.so locally
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/__init__.py", line 23, in <module>
    from tensorflow.python import *
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/__init__.py", line 53, in <module>
    from tensorflow.core.framework.graph_pb2 import *
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/core/framework/graph_pb2.py", line 9, in <module>
    from google.protobuf import symbol_database as _symbol_database
  File "/usr/local/lib/python2.7/dist-packages/google/protobuf/symbol_database.py", line 164, in <module>
    _DEFAULT = SymbolDatabase(pool=descriptor_pool.Default())
AttributeError: 'module' object has no attribute 'Default'
```
How to solve this problem? Thanks.

**Comment by aselle** [comment]
Looks like protobufs is having trouble loading. Make sure you are running pb3... i.e.
```pip install --upgrade protobu```.

**Comment by CTTC** [comment]
@aselle Thanks for your reply. And I have removed the original protobuf 2 and installed the latest protobuf 3.1.0 from source. But I still get the same error. What might be wrong?

**Comment by CTTC** [comment]
@aselle  And I used this whl:
```bash
https://storage.googleapis.com/tensorflow/linux/gpu/tensorflow-0.11.0-cp27-none-linux_x86_64.whl
```

**Comment by CTTC** [comment]
I have solved my problem. It turns out that even though I  upgraded to the latest protobuf 3.1.0, there are still some remaining protobuf 3.0.0 files in my system. More specifically, protobuf 3.1.0 was installed to /usr/local/lib/python2.7/dist-packages, and the original protobuf 3.0.0 was installed in /usr/lib/python2.7/dist-packages. After I manually deleted those files, I can import tensorflow.

**Comment by AntoineHus** [comment]
Hello Guys, Thank you, it solved my issue with tensorflow 0.11 and my own compilation with gpu.

**Comment by sallymmx** [comment]
@CTTC  I have also solved my problem by your method, thanks!
